content/en/observability_pipelines/troubleshooting.md

```diff
@@ -43,8 +43,31 @@ observability-pipelines-worker tap <component_ID>
 
 If you are using a containerized environment, use the `docker exec` or `kubectl exec` command to get a shell into the container to run the above `tap` command.
 
+## Seeing delayed logs at the destination
+
+Observability Pipelines destinations batch events before sending them to the downstream integration. For example, the Amazon S3, Google Cloud Storage, and Azure Storage destinations have a batch timeout of 900 seconds. If the other batch parameters (maximum events and maximum bytes) are not met, the batch is flushed at the 900-second timeout. This means the destination component can take up to 15 minutes to send out a batch of events to the downstream integration.
```

Comment:
```suggestion
Observability Pipelines destinations batch events before sending them to the downstream integration. For example, the Amazon S3, Google Cloud Storage, and Azure Storage destinations have a batch timeout of 900 seconds. If the other batch parameters (maximum events and maximum bytes) have not been met by the 900-second timeout, the batch is flushed because it has reached the timeout. This means the destination component can take up to 15 minutes to send out a batch of events to the downstream integration.
```